client/src/components/Workflow/Editor/Attributes.vue

```diff
@@ -105,7 +105,7 @@ export default {
             default: null,
         },
         parameters: {
-            type: LegacyParameters,
+            type: ImplicitParameters,
```

Comment:
Call it LegacyWorkflowParamater maybe ? I don't think anything about this is implicit.